es-vue-base/tests/EsPagination.spec.js

```diff
@@ -0,0 +1,20 @@
+import { mount } from '@vue/test-utils';
+import EsPagination from '@/src/lib-components/EsPagination.vue';
+import jestVue from '@/tests/jest.vue.config';
+
+describe('EsPagination', () => {
```

Comment:
Since the control of what's rendered is not part of the pagination component itself, I'm not sure this would work to write tests for in the same way - `currentPage` is modified by pagination, but after that it's up to the list (or table etc.) to selectively render:

```<li
    v-for="item in items.slice(perPage * (currentPage - 1), perPage * currentPage)"
    :key="item">{{ item }}</li>```